packages/datetime/src/dateRangePicker.tsx

```diff
@@ -35,6 +35,15 @@ import { TimePicker } from "./timePicker";
 export interface IDateRangeShortcut {
     label: string;
     dateRange: DateRange;
+
+    /**
+     * By default, clicking a shortcut does not change the time of the date range picker, but instead
+     * takes the date components of the `dateRange` and combines it with the currently selected time.
+     * Setting `shouldChangeTime` to `true` will override this behavior and allow shortcuts to change
+     * the selected time as well.
+     * @default false
+     */
+    shouldChangeTime?: boolean;
```

Comment:
thoughts on calling this `includeTime`? clarifies that the default behavior ignores time components and you must enable this field to respect them.

Reply:
Though prop name is `shouldChangeTime` in PR, it was indeed changed to `includeTime` somewhere later (for those who will try it)